src/features/basic-resolve.js

```diff
@@ -1,11 +1,13 @@
 import { systemJSPrototype } from '../system-core.js';
 import { resolveIfNotPlainOrUrl, baseUrl } from '../common.js';
+import { errMsg } from '../err-msg.js';
+
 systemJSPrototype.resolve = function (id, parentUrl) {
   const resolved = resolveIfNotPlainOrUrl(id, parentUrl || baseUrl);
   if (!resolved) {
     if (id.indexOf(':') !== -1)
       return Promise.resolve(id);
-    throw Error('Cannot resolve "' + id + (parentUrl ? '" from ' + parentUrl : '"'));
+    throw Error(errMsg(3, DEV ? 'Cannot resolve "' + id + (parentUrl ? '" from ' + parentUrl : '"') : [id, parentUrl].join(', ')));
```

Comment:
I've just removed this error in the other PR, which should simplify things a bit (only one "cannot resolve" now). Perhaps we can remove the "from import map" specifics in the message too?

Reply:
Better might be - `Cannot resolve bare specifier`